<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_principal</class>
 <widget class="QMainWindow" name="vtn_principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ingreso Persona</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_nombre">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Nombre:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_nombre">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>40</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_grabar">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>390</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>Grabar</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_apellido">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <pointsize>8</pointsize>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Apellido:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_apellido">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="maxLength">
     <number>50</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_persona">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Docente</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Estudiante</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="lbl_tipo_persona">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Tipo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_cedula">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Cedula:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_cedula">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="maxLength">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_email">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Email:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_email">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_buscar_cedula">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>Buscar por cedula</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_estatura">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Estatura(cm)l:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_estatura">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>330</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>300</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_estatura">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>330</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>E.D. Estatura</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_peso">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>360</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Peso(kg):</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_peso">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>90</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_edad">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Edad:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sp_edad">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>300</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_edad_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Fecha de Nacimiento:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_edad">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>300</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>E.D. EDAD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_peso">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>E.D. PESO</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_carrera">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_carrera">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Carrera:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dae_fecha_nacimiento">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>270</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_tipo_persona_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Tw Cen MT Condensed Extra Bold</family>
      <underline>true</underline>
     </font>
    </property>
    <property name="text">
     <string>Habilitado:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_tipo_habilitado">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>240</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <item>
     <property name="text">
      <string>No</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Si</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btn_cal_edad">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>270</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Bahnschrift Light SemiCondensed</family>
     </font>
    </property>
    <property name="text">
     <string>C.Edad</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mnb_menu_principal">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>324</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="stb_estado"/>
 </widget>
 <resources/>
 <connections/>
</ui>
